src/rust/generators/sysmon-subgraph-generator/benches/generator_bench.rs

```diff
@@ -0,0 +1,62 @@
+#![cfg(test)]
+#![feature(test)]
+
+extern crate test;
+
+use std::io::Write;
+
+use grapl_test::cache::EmptyCache;
+use sqs_executor::{event_decoder::PayloadDecoder,
+                   event_handler::CompletedEvents};
+use sysmon_subgraph_generator_lib::{generator::SysmonSubgraphGenerator,
+                                    metrics::SysmonSubgraphGeneratorMetrics,
+                                    serialization::SysmonDecoder};
+use test::Bencher;
+use tokio::runtime::Runtime;
+
+const SYSMON_SAMPLE_DATA_FILE: &'static str = "../../../../etc/sample_data/events6.xml";
```

Comment:
I know we're mono repo, but in the future I'd like to see this kind of thing be more self contained. In this case, that'd be moving to events6.xml to the sysmon generator directory. It's not like that file is useful without that generator.

Not to say I wanna require that for this PR, but  if you agree I'll bring it up for discussion.

Reply:
Yeah, I think we should use a dedicated benchmark file - events6.xml could move, or change in the future.

Reply:
Will move a copy of the test data into the repos specifically! Sorry; got lazy during hack week :)

The code limits the number of parsed events that each generator is processing so the results should be relatively consistent and comparable.